cleanlab/filter.py

```diff
@@ -339,6 +341,15 @@ def find_label_issues(
     verbose : int
       If 0, no print statements. If 1, prints when multiprocessing happens.
 
+    rank_by_kwargs : dict
+      Optional keyword arguments to pass into scoring functions for ranking.
+      Accepted args include:
+        adj_pred_probs : bool, default = False
+          Adjust predicted probabilities by subtracting the class confident thresholds and renormalizing.
+          The confident class threshold for a class j is the expected (average) "self-confidence" for class j.
+          See paper "Confident Learning: Estimating Uncertainty in Dataset Labels" by Northcutt et al.
+          https://arxiv.org/abs/1911.00068
```

Comment:
would specify exactly which method these kwargs will be passed into. Then refer user to that method's documentation rather than trying to explain all the kwargs here. This reduces redundancy in the docstrings.

Reply:
Agree. Done, thanks